File: Projects/server/Contracts/Requests/StartTripRequest.cs
namespace server.Contracts.Requests
{
    public class StartTripRequest
    {
        public DateTime StartDatetime { get; set; }
        public string MacAddress { get; set; } = null!;
        public uint CarId { get; set; }
        public byte[] EcuId { get; set; } = null!;
        public uint Supported { get; set; }
    }
}

File: Projects/api_test_app_rest_assured/src/test/java/models/requests/StartTripRequest.java
package models.requests;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Setter
@Getter
@Builder
public class StartTripRequest {
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
    private LocalDateTime startDateTime;
    private String macAddress;
    private Integer carId;
}
